bazel/ngraph.BUILD

```diff
@@ -285,3 +303,46 @@ cc_library(
     alwayslink = 1,
 )
 
+cc_library(
```

Comment:
The interpreter backend wasn't needed but since you've done it let's keep it. However, the target doesn't have the same flags - so please add them. Also - please ensure that it's tested.